joomla/joomla-cms
# Regression: Messages paragraph spacing

#### Summary of Changes

This PR partly revert the changes made in PR https://github.com/joomla/joomla-cms/pull/9181

Unfortunately we had some side effect after that PR: When using messages with several "p" tags the line spacing has also lost. Two of those examples are the Stats notification and the Post install messages as show in the image below.
![image](https://cloud.githubusercontent.com/assets/9630530/13355382/f7186cdc-dc97-11e5-8318-3843fb71f4a7.png)

This PR, besides reverting the "p" tag spacing, also changes the message container from a "p" tag to a "div" tag. This will also allow to had block level HTML elements inside the message ("p", "ul", "ol" etc).

After applying the patch you will see all message as before and this "special" messages with the correct spacing between paragraph as one can see in the image below.

![image](https://cloud.githubusercontent.com/assets/9630530/13355454/6999ccd8-dc98-11e5-8466-f5b9420fb0eb.png)
#### Testing Instructions
1. Use latest staging.
2. Enable stats plugin and reset the unique id, so the message appears.
3. Go to control panel and check the paragraph spacing.
4. Apply this PR and clean browser cache (Ctrl+F5)
5. Repeat step 3. You will see the message is now with the correct spacing between paragraphs.
6. Check other info, warning, sucess and error messages in the frontend and in the backend.
